db/seeds.rb

```diff
@@ -69,7 +73,8 @@
   reason_to_donate: 'I believe in fighting bigotry',
   slug: 'southern-poverty-law-center',
   info_url: 'http://www.splcenter.org/',
-  donation_url: 'https://donate.splcenter.org/sslpage.aspx?pid=463'
+  donation_url: 'https://donate.splcenter.org/sslpage.aspx?pid=464',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.